src/etc/init.d/load-modules

```diff
@@ -7,7 +7,7 @@ loadmods() {
 	while read line; do
 		line="${line%%#*}"
 		[ -n "$line" ] || continue
-		modprobe $line
+		modprobe -q $line
```

Comment:
but doesn't this just silence all errors?

I'd like to ask the kernel for a given functionality "driver for virtio-blk" or "vfat filesystem support" and if that functionality isn't there, then warn that it failed and we wont have it. Maybe thats not possible.

Reply:
Ok. Will be more complicated then.

Reply:
yeah... so maybe this is fine.
but I want some way to check if the system looks fine.

i dont like the warnings, but I dont want to just hide them all the time.
because doing that means its hard to tell when we made a mistake or something changed and we lost support for something (module rename or something).

thoughts?

Reply:
Let's go with my 2nd version - it checks first. 

But it also looks like initramfs is using different script to load modules as test boot shows:

```
info: initramfs: up at 2.61
modprobe: module qemu_fw_cfg not found in modules.dep
modprobe: module virtio_blk not found in modules.dep
modprobe: module virtio_net not found in modules.dep
modprobe: module virtio_pci not found in modules.dep
modprobe: module vfat not found in modules.dep
modprobe: module nls_cp437 not found in modules.dep
modprobe: module nls_iso8859-1 not found in modules.dep
modprobe: module cdrom not found in modules.dep
modprobe: module sr_mod not found in modules.dep
modprobe: module sg not found in modules.dep
info: copying initramfs to /dev/vda1
info: initramfs loading root from /dev/vda1
info: /etc/init.d/rc.sysinit: up at 5.01
info: container: none
modprobe: module qemu_fw_cfg not found in modules.dep
modprobe: module vfat not found in modules.dep
modprobe: module nls_cp437 not found in modules.dep
modprobe: module nls_iso8859-1 not found in modules.dep
modprobe: module cdrom not found in modules.dep
WARN: /etc/rc3.d/S10-load-modules failed
```

Reply:
Same script. Just no /sys/ mounted when called first time.